lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,44 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Tab = props => {
+ const Tab = props => {
+  if (this.props.tab === selectedTab) {
```

Comment:
The way that you are accessing props here is incorrect - think about the type of component that you have. Additionally, selectedTab is undefined. Where does selectedTab live?